repose-aggregator/core/repose-core-api/src/main/resources/META-INF/schema/container/container-configuration.xsd

```diff
@@ -252,6 +257,55 @@
         </xs:simpleContent>
     </xs:complexType>
 
+    <xs:complexType name="ViaConfiguration">
+        <xs:annotation>
+            <xs:documentation>
+                <html:p>
+                    Contains the information needed to configure Repose's Via header.
+                </html:p>
+            </xs:documentation>
+        </xs:annotation>
+        <xs:simpleContent>
+            <xs:extension base="papic:EmptyString">
+                <xs:attribute name="enabled" type="xs:boolean" use="optional" default="true">
+                    <xs:annotation>
+                        <xs:documentation>
+                            <html:p>Indicates whether or not the Via header is created.</html:p>
+                        </xs:documentation>
+                    </xs:annotation>
+                </xs:attribute>
+                <xs:attribute name="content" type="xs:string" use="optional">
```

Comment:
I think these attributes are slightly different than discussed.

So it looks to me like `content` will be the full value of the `Via` header. Is that correct? If so, I believe we only want it to be part of the `Via` header, the way the custom, configurable part of the `Via` header is now. That may just be a matter of naming.

The `enabled` attribute also seems to indicate that it turns `Via` header creation on/off. I believe what it's supposed to do is turn the `(Repose/99.99)` part of the `Via` header on/off. Again, that may just be in the name.

So with these attributes, I _believe_ I can configure the following cases:
* No Via header created
* Via header created with Repose value
* Via header created with custom value

However, the following case is not supported, but should be:
* Via header created with Repose and custom value combined

The way I would achieve that is by having your boolean attribute dictate whether or not to include the `(Repose/99.99)` bit, and the string attribute dictate whether or not to add a prefix. Then, to disable the via header, I would simply set the boolean attribute to false and omit the string attribute. To send only the Repose portion, I would set the boolean attribute to true and omit the string attribute. To send only custom content, I would set the boolean attribute to false and configure the string attribute. To send a mix of content, I would set the boolean value to true and configure the string attribute.